### ISSUE TITLE
[tag retention] tags and labels are meaningless when set "retain none"

### ISSUE THREAD

**Issue by wy65701436** [goharbor/harbor#8695]
![image](https://user-images.githubusercontent.com/2841473/63079534-2eddac80-bf71-11e9-935a-cc24df254013.png)

**Comment by steven-zou** [comment]
I don't think so. The meaning is `Delete all the matched artifacts ` ("DO NOT retain some of the artifacts")

The selectors can be used to narrow down the scope of `retain none`

**Comment by xaleeks** [comment]
It is a little confusing.  Just to make sure, 'retain none matching *prod-' right now will have the same effect as 'delete *prod-' ?  So retain now is essentially delete?

**Comment by bitsf** [comment]
"retain none" will delete all matching repositories, no matter what tags or labels are.

**Comment by bitsf** [comment]
per discuss
1. remove "retain none"
2. change "retain always" to "retain"
3. sort rule

**Comment by bitsf** [comment]
fix by https://github.com/goharbor/harbor/pull/8763
